kenshin-profile-app/kenshin-profile-app/SNS.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  SNS.swift
+//  kenshin-profile-app
+//
+//  Created by miyamotokenshin on R 7/06/09.
+//
+
+import SwiftUI
+
+struct SNS: View {
```

Comment:
Viewだから最後にViewをつけたほうがいいね

```suggestion
struct SNSView: View {
```

Reply:
済